pkg/input/container/docker.go

```diff
@@ -146,7 +146,9 @@ func (dt *DockerTailer) readForever() {
 		inBuf := make([]byte, 4096)
 		n, err := dt.reader.Read(inBuf)
 		if err == io.EOF {
-			dt.wait()
+			// reader is closed, maybe container stopped running
```

Comment:
Does that change means tailing a log that is slowly being appended to will open/close it every time ? Is it efficient ?

Reply:
From my tests, `EOF` happens only when the stream is closed (ie container stops), not when we've read all there is to read.